SPECS/p11-kit/p11-kit.spec

```diff
@@ -1,15 +1,15 @@
 %define _userunitdir /usr/lib/systemd/user
 
-Version:        0.23.16.1
-Release:        2%{?dist}
+Version:        0.23.22
```

Comment:
Looks good, but I'd like to ask you to test something before I approve. Could you rebuild the `ca-certificates` package with these changes, build a new image (VHD, VHDX, ISO, doesn't matter) and make sure that:
1. DNF/TDNF can install something from our repo (for instance `ca-certificates`, because by default we should only have the base), and
2. after installing `ca-certificates` the command `curl https://www.google.com` succeeds? It should fail with only the base set of certificates.

Reply:
I built a VHD and verified:

1) TDNF can install a package. I tried our `ca-certificates` packages (e.g. msft, legacy, etc.) and `python2`.
2) `curl https://www.bing.com` succeeds